### ISSUE TITLE
Fuzzing binaries with LLVM's libFuzzer and rev.ng

### ISSUE THREAD

**Issue by edi33416** [revng/revng#57]
Hello,

I'm trying to reproduce the steps presented in the [Fuzzing binaries with LLVM's libFuzzer and rev.ng](https://rev.ng/blog/fuzzing/post.html) blog post, but I wasn't able to find the LLVM passes mentioned in the blog post.

Are the passes mentioned in the post available on GitHub?
How would one go about to reproduce the steps described in the blog post?

Looking forward to your answer,
Eduard

**Comment by aleclearmind** [comment]
Hi Eduard, we're looking into making some of that available.
We want to push out the part enabling to use `libFuzzer` on lifted binaries soon.
But the part where you can call directly functions with a nice prototype has been postponed after we finalize #27 and #37.

Will post an update once the PR for using `libFuzzer` is ready.

**Comment by edi33416** [comment]
Thank you for the updates and for the guidelines. I'll look into it

**Comment by aleclearmind** [comment]
Fuzzing is no longer one of our core goals. Closing this.
